src/Microsoft.DotNet.ProjectJsonMigration/MigrationTrace.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+  public class MigrationTrace
```

Comment:
ahh, project ryder was doing this for a little bit. Thought I'd got them all, fixed.